metricbeat/module/system/process/process.go

```diff
@@ -65,6 +65,10 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		}
 	}
 
+	if config.Pid != 0 && config.Procs[0] != ".*" {
+		logp.L().Warnf("`processes.pid` set to %d, but `processes` is set to a non-default value. Metricset will only report metrics for pid %d", config.Pid, config.Pid)
```

Comment:
`processes.pid` -> `process.pid` (or just `pid` if you take the other suggestion)